kore/src/Kore/Internal/Conditional.hs

```diff
@@ -233,9 +228,9 @@ instance
 
 instance
     InternalVariable variable
-    => From (SingleSubstitution variable) (Conditional variable ())
+    => From (Assignment variable) (Conditional variable ())
   where
-    from = from @(Substitution variable) . from
+    from = from @(Substitution variable) . from . Substitution.assignmentToPair
```

Comment:
Wouldn't there be an `instance (Assignment variable) (Substitution variable)` so that `Substitution.assignmentToPair` isn't necessary here?